.github/workflows/git-artifacts.yml

```diff
@@ -96,26 +100,34 @@ jobs:
           name: bundle-artifacts
           path: bundle-artifacts
   pkg:
-    runs-on: windows-latest
+    # Hack to ensure that the "determine skip" step works if no self-hosted ARM64 runner is available
+    runs-on: ${{ ((matrix.arch.name == 'aarch64' && github.event.inputs.build_arm64 == 'true') && fromJSON('["Windows", "ARM64"]')) || 'windows-latest' }}
```

Comment:
The `fromJSON()` part has been a pain to get right (and [I'm not the only one](https://github.com/github/docs/issues/20495)), but it works now.

In any case, I think it'll be more scalable to set the entire `matrix` dynamically ([example here](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson)). That also allows us to get rid of all those `if: env.SKIP != 'true'` conditions in the pipeline. But let's leave that for a follow-up PR, as I spent a _lot_ of time getting all of this to work 🙏🏼

Reply:
> I think it'll be more scalable to set the entire matrix dynamically ([example here](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson)).

We also have a splendid example here: https://github.com/git-for-windows/build-extra/blob/f6041a722c0cd6a36d139537f1253aaae74800e9/.github/workflows/main.yml#L19-L24 (and here: https://github.com/git-for-windows/build-extra/blob/f6041a722c0cd6a36d139537f1253aaae74800e9/.github/workflows/main.yml#L47).